.ci/pipeline/docs.yml

```
@@ -54,6 +54,38 @@ jobs:
     inputs:
       versionSpec: '3.11'
       addToPath: true
+  - script: |
+      # Extract the version from the triggering tag
+      DOC_VERSION=$(git describe --tags --abbrev=0)
+      echo "DOC_VERSION1: $DOC_VERSION"
+
+      # If no tag is found, fetch the latest version from PyPI
```

I don't think we'd want to fetch something from PyPI for the purpose of building new docs. Better to let it fail.

---

Sure, that make sense, I removed this part now it will fail if cannot find tag Best, Yue